src/test/ui/issue-49040.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![allow(unused_variables)];
```

Comment:
Error annotation in the test itself is also required - `//~ ERROR unexpected semicolon`.

It also looks you have composed the `.stderr` file manually (stderr files don't have licenses), the script `src/test/ui/update-all-references.sh` can be used to generate it automatically.

CI currently fails due to these two issues.